### ISSUE TITLE
[BUG] Invalid classnames-order on daisyui dropdown

### ISSUE THREAD

**Issue by Aragur** [francoismassart/eslint-plugin-tailwindcss#268]
**Describe the bug**

**To Reproduce**
1. Install [DaisyUI](https://daisyui.com/)
2. Create a [Navbar including a Dropdown](https://daisyui.com/components/navbar/)
3. The classes`dropdown-end` and `dropdown` each want to be at the front. A warning is always generated.

**Expected behavior**
A clearly defined order should be followed.

**Screenshots**
<img width="630" alt="Screenshot 2023-08-22 at 18 20 33" src="https://github.com/francoismassart/eslint-plugin-tailwindcss/assets/11004008/5b2a30fc-41a8-4b35-8df1-20e04d606773">

**Environment (please complete the following information):**
 - OS: macOS
 - Softwares + version used:
     -  VSCode 1.81.1

**Additional context**
None

**eslint config file or live demo**
```html
<div class="navbar bg-base-100">
  <div class="flex-1">
    <a class="btn btn-ghost normal-case text-xl">daisyUI</a>
  </div>
  <div class="flex-none gap-2">
    <div class="form-control">
      <input type="text" placeholder="Search" class="input input-bordered w-24 md:w-auto" />
    </div>
    <div class="dropdown dropdown-end">
      <label tabindex="0" class="btn btn-ghost btn-circle avatar">
        <div class="w-10 rounded-full">
          <img src="/images/stock/photo-1534528741775-53994a69daeb.jpg" />
        </div>
      </label>
      <ul tabindex="0" class="mt-3 z-[1] p-2 shadow menu menu-sm dropdown-content bg-base-100 rounded-box w-52">
        <li>
          <a class="justify-between">
            Profile
            <span class="badge">New</span>
          </a>
        </li>
        <li><a>Settings</a></li>
        <li><a>Logout</a></li>
      </ul>
    </div>
  </div>
</div>
```

**Comment by kachkaev** [comment]
👋 @Aragur not sure what the issue is. If you reorder class names as the rule suggest, what happens? There is a _Quick fix_ button in the popover on your screenshot.

**Comment by Aragur** [comment]
@kachkaev When I click on the `Quick fix` button the order changes to ```dropdown-end dropdown```, but this is also marked as being wrong → so there's an infinite loop.

**Comment by Jaynarol** [comment]
I have same problem with  className="swap-rotate swap" from daisyui too. it infinite loop quick fix between "swap-rotate swap" and "swap swap-rotate"

**Comment by francoismassart** [comment]
Closing this issue, cleaning up before release of v4.
